### PR TITLE
Chore: use next/link in nav

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Use next/link in nav for better SEO and other benefits.

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/www/components/Nav/Developers.tsx [hunk 1]
```diff
@@ -37,13 +38,11 @@ const Developers = () => {
...
     )
     return url ? (
-      <a
-        key={text}
-        href={url}
-        className="dark:hover:bg-scale-500 col-span-6 rounded p-3 transition hover:bg-gray-50"
-      >
-        {content}
-      </a>
+      <Link href={url} key={text}>
```
Comment (MildTomato) [inline apps/www/components/Nav/Developers.tsx:41]: might need to use `passHref` here

Comment (MildTomato) [reply apps/www/components/Nav/Developers.tsx:41]: https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

File: apps/www/components/Nav/Developers.tsx [hunk 2]
```diff
@@ -65,25 +64,24 @@ const Developers = () => {
...
-                    />
-                  </div>
-                  <div className="min-w-0 flex-1 sm:ml-4">
-                    <h4 className="text-scale-1200 text-normal mb-0 text-base">
-                      {caseStudy.title}
-                    </h4>
-                    <p className="p text-sm">{caseStudy.description}</p>
-                  </div>
-                </a>
+                <Link href={caseStudy.url}>
```
Comment (MildTomato) [inline apps/www/components/Nav/Developers.tsx:67]: same here

**Review by MildTomato — CHANGES_REQUESTED** [review]
just need `passHref` i think.

Looks good other than that 🔥

File: apps/www/components/Nav/Product.tsx [hunk 3]
```diff
@@ -30,13 +31,11 @@ const Product = () => {
...
     return (
       url && (
-        <a
-          key={name}
-          href={url}
-          className="hover:bg-scale-300 dark:hover:bg-scale-500 col-span-6 rounded p-3 transition"
-        >
-          {content}
-        </a>
+        <Link href={url} key={name}>
```
Comment (MildTomato) [inline apps/www/components/Nav/Product.tsx:34]: here

File: apps/www/components/Nav/Product.tsx [hunk 4]
```diff
@@ -56,25 +55,24 @@ const Product = () => {
...
-                      />
-                    </div>
-                    <div className="min-w-0 flex-1 sm:ml-4">
-                      <h4 className="text-scale-1200 text-normal mb-0 text-base">
-                        {caseStudy.title}
-                      </h4>
-                      <p className="p text-sm">{caseStudy.description}</p>
-                    </div>
-                  </a>
+                  <Link href={caseStudy.url}>
```
Comment (MildTomato) [inline apps/www/components/Nav/Product.tsx:58]: here

File: apps/www/components/Nav/index.tsx [hunk 5]
```diff
@@ -84,13 +84,11 @@ const Nav = () => {
...
     )
     return url ? (
-      <a
-        key={`solution_${idx}`}
-        href={url}
-        className="dark:hover:bg-scale-600 -m-3 my-2 flex flex-col justify-between rounded-lg p-3 transition duration-150 ease-in-out hover:bg-gray-50"
-      >
-        {content}
-      </a>
+      <Link href={url} key={`solution_${idx}`}>
```
Comment (MildTomato) [inline apps/www/components/Nav/index.tsx:87]: here

File: apps/www/components/Nav/index.tsx [hunk 6]
```diff
@@ -218,26 +216,28 @@ const Nav = () => {
...
+                      className={`
+                        text-scale-1200 hover:text-brand-900 hover:border-brand-900 dark:text-dark-100 dark:hover:border-dark-100 inline-flex items-center
+                        border-b-2 border-transparent p-5 px-1
+                        text-sm font-medium
+                      `}
+                    >
+                      Pricing
+                    </a>
+                  </Link>
+                  <Link href="/blog">
```
Comment (MildTomato) [inline apps/www/components/Nav/index.tsx:230]: here

File: apps/www/components/Nav/index.tsx [hunk 7]
```diff
@@ -218,26 +216,28 @@ const Nav = () => {
...
-                    href="/blog"
-                    className={`
-                    text-scale-1200 hover:text-brand-900 hover:border-brand-900 dark:text-dark-100 dark:hover:border-dark-100 inline-flex items-center
-                    border-b-2 border-transparent p-5 px-1
-                    text-sm font-medium
-                  `}
-                  >
-                    Blog
-                  </a>
+                  <Link href="/pricing">
```
Comment (MildTomato) [inline apps/www/components/Nav/index.tsx:219]: here

File: apps/www/components/Nav/index.tsx [hunk 8]
```diff
@@ -270,14 +270,18 @@ const Nav = () => {
...
 
-                <a href="https://app.supabase.io/">
-                  <Button type="default" className="hidden lg:block">
-                    Sign in
-                  </Button>
-                </a>
-                <a href="https://app.supabase.io/">
-                  <Button className="hidden text-white lg:block">Start your project</Button>
-                </a>
+                <Link href="https://app.supabase.io/">
```
Comment (MildTomato) [inline apps/www/components/Nav/index.tsx:273]: here

File: apps/www/components/Nav/index.tsx [hunk 9]
```diff
@@ -270,14 +270,18 @@ const Nav = () => {
...
+                  <a>
+                    <Button type="default" className="hidden lg:block">
+                      Sign in
+                    </Button>
+                  </a>
+                </Link>
+                <Link href="https://app.supabase.io/">
```
Comment (MildTomato) [inline apps/www/components/Nav/index.tsx:280]: here

File: apps/www/components/Nav/index.tsx [hunk 10]
```diff
@@ -324,46 +328,45 @@ const Nav = () => {
               {/* </div> */}
               <div className="mt-6 mb-12">
                 <div className="space-y-1 pt-2 pb-4">
-                  <a
-                    href="https://app.supabase.io/"
-                    className="text-scale-900 block pl-3 pr-4 text-base font-medium dark:text-white"
-                  >
-                    Sign in
-                  </a>
+                  <Link href="https://app.supabase.io/">
```
Comment (MildTomato) [inline apps/www/components/Nav/index.tsx:331]: all of these and below :p

**Review by MildTomato — CHANGES_REQUESTED** [review]
😛  hehe

**Comment by Isaiah-Hamilton** [comment]
@MildTomato I'm not sure that all these `passHerf` are necessary because `passHerf` is only required with custom components that wraps an `<a>` tag.

**Comment by MildTomato** [comment]
@Isaiah-Hamilton ahh sorry you're right. revert all my requests 🙈 sorry about that. misread the docs

**Review by MildTomato — APPROVED** [review]

**Pull request merged**